README.md

```diff
@@ -1 +1,36 @@
-# Conference
+# Marketing Conference
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+Created a global marketing conference website. It has Home Page and About Page and also added the user interaction link, mobile menu, Implement the section "Featured speakers" with dynamic HTML.
+In mobile version when 'MORE' button is dynamically used and followed the design.
+
+
+
+## Built With
+
+- HTML & CSS
+- Java Script
+- linters
+
+## Live Demo
+[Live Demo Link](https://monika-soam.github.io/Marketing-Conference/)
+
+ 
+
+👤 **Monika**
+
+- GitHub: https://github.com/monika-soam
+- Twitter: @monikasoam
+- LinkedIn: linkedin.com/in/monika-soam-✓-16b59925
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project! and advice me where to work upon it
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
Kindly add an "Acknowledgements" to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/)

Reply:
This is fixed